packages/devextreme/scss/widgets/base/_form.scss

```diff
@@ -144,7 +136,7 @@
 
   .dx-form-group {
     &.dx-form-group-with-caption {
-      padding-left: 20px;
+      padding-inline-start: 20px;
```

Comment:
Are you sure you shouldn't use variables here?

Reply:
There are a lot of constant values uses in form styles. I suggest to extract these values to variables in a separate task.